.dockerignore

```diff
@@ -0,0 +1,11 @@
+### macOS ###
+.DS_Store
+
+### Virtual environment ###
+env/
+.venv/
+
+### Configuration files ###
+.env
+### module4
+.vagrant
```

Comment:
This is the right idea but more things could be added. E.g `Vagrantfile`, `.git`, `.gitignore`, `__pycache__` `Dockerfile`